src/app/admin/createUser/CreateUserForm.tsx

```diff
@@ -13,8 +13,10 @@ import {
 import { createUser } from "@/app/admin/adminActions";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faUserPlus } from "@fortawesome/free-solid-svg-icons";
-import RefreshPageButton from "@/app/admin/RefreshPageButton";
+import RefreshPageButton from "@/app/admin/common/RefreshPageButton";
```

Comment:
Idk if this is just me but I feel like common could be changed to like sharedComponents or somethign. But also that is again me being pedantic, feel free to ignore.